src/templates/global/authenticate_message.html.tt

```diff
@@ -81,7 +81,10 @@
         l('Only JPG, JPEG, GIF, PNG and PDF formats are accepted'), 
         l('Maximum upload size for each file is 3MB')]
     type="poi"
-    accepted_documents=[l('Passport'), l('Identity card'), l('Driving license')]
+    accepted_documents=[{name => l('Passport'), value => 'passport'},
```

Comment:
Nothing serious, but appreciate it if you move the first index to next line and vertically align keys for better readability.
Thanks.